core/src/com/mygdx/game/ecs/systems/BotAttackSystem.java

```diff
@@ -23,7 +31,14 @@ protected void processEntity(Entity entity, float deltaTime) {
         long currentTime = (new Date()).getTime();
 
         if (botComponent.attacking && ((currentTime - botComponent.lastAttack) > botComponent.attackInterval)) {
-            // TODO: decrease health in database here
+            // Creating a random attack value to create a more dynamic game experience
+            int randomAttackValue = ThreadLocalRandom.current().nextInt(2,7);

```

Comment:
Hadde vært kult om attackPower til zombien blir brukt for å beregne denne. F
Eks. attackPower +- 3 ellerno ^^
Da er det også lagt mer opp til andre typer fiender som kan ta mer eller mindre liv 👌

Reply:
oohh hadde nok vært nice. Men tror ikke jeg orker å endre på det. Kan heller gjøre det om vi ser vi har ekstra tid mot slutten av uka.